codespell_lib/_codespell.py

```diff
@@ -897,6 +897,19 @@ def parse_file(
             word = match.group()
             lword = word.lower()
             if lword in misspellings:
+                # Sometimes we find a 'misspelling' which is actually a valid word
+                # preceeded by a string escape sequence.  Ignore such cases as
```

Comment:
Hah, `codespell` got you :)
```suggestion
                # preceded by a string escape sequence.  Ignore such cases as
```